examples/basics/README.md

```diff
@@ -44,17 +44,29 @@ The example read_EDM4HEP.py shows you how to access the different objects such a
 `dataframe.Define("<your_variable>", "<accessor_fct (<name_object>)>")`
 which creates a column in the RDataFrame named `<your_variable>` and filled with the return value of the `<accessor_fct>` for the given object. 
 
-Here, accessor functions are the functions found in the C++ analyzers code that return a certain variable, e.g. getRP_pt(object) returns the pT of the object. To find out all functions available for e.g. reconstructed particles have a look at the [respective analyzers code](https://github.com/bistapf/FCCAnalyses/blob/basicexamples/analyzers/dataframe/ReconstructedParticle.h). If you want to add your own function have a look at the [Writing your own function](#writing-your-own-function) section on this page. 
+Here, accessor functions are the functions found in the C++ analyzers code that return a certain variable. Since the analyzers code defines a specific namespace for each module, such as ReconstructedParticle or MCParticle, the full accessor function call looks like `<namespace>::<function_name>(object)`. To access the pT of a reconstructed object you would therefore call `ReconstructedParticle::get_pt(object)` and for a MC-particle the call would be `MCParticle::get_pt(object)`. The namespace corresponds to the file name of the C++ code, making it clear where to look for the source code if you have a question about the internal workings of one such functions. 
 
-For the name of the object, in principle the names of the EDM4HEP collections are used - photons, muons and electrons are an exception where a few extra steps are required, as shown in the example here. 
+Below you can find an overview of the basic, most commonly required functions, to illustrate the naming conventions. This is not an exhaustive list, if you want to find out all functions that are available please take a look in the respective analyzers code itself - [here for reconstructed particles](https://github.com/bistapf/FCCAnalyses/blob/basicexamples/analyzers/dataframe/ReconstructedParticle.h) and [here for MC particles](https://github.com/bistapf/FCCAnalyses/blob/basicexamples/analyzers/dataframe/MCParticle.h).
```

Comment:
Here there two links to a prvate repository (bistapf). Can these point to the files in the official directory? (If there are changes to those files, the changes should be perhaps parts of the PR?)

Reply:
I can fix the links after the merge.